notifications/register/create-binding-server/create-binding-server-apn.5.x.rb

```diff
@@ -0,0 +1,23 @@
+# Download the Ruby helper library from twilio.com/docs/libraries/ruby
+require 'twilio-ruby'
+
+# Get your Account Sid and Auth Token from https://www.twilio.com/console
+account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
+auth_token = 'your_auth_token'
+
+# set up a client to talk to the Twilio REST API
+client = Twilio::REST::Client.new(account_sid, auth_token)
+
+# Retrieve a valid notification service
+service = client.notify.v1.services('ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
+
+# Create a binding for a given identity
```

Comment:
> "A comment is a lie waiting to happen."

It's true that this code creates a `binding` for a given identity, but it is also true that this binding depends on various parameters. We also consider parameters like the `endpoint`, `binding_type` and so on.

I don't think this comment is adding value to the example. What about removing it?